RPM/devel/zlib-devel/rpmbuild/SPECS/zlib-devel.spec

```diff
@@ -19,27 +19,30 @@ to develop programs that use the zlib compression and decompression
 library.
 
 %install
-mkdir -p %{buildroot}/lib64/pkgconfig
-mkdir -p %{buildroot}/usr/include
-mkdir -p %{buildroot}/share/man/man3/
-cp /lib64/libz.a %{buildroot}/lib64/
-ln -s /lib64/libz.so.1.2.11 %{buildroot}/lib64/libz.so
-ln -s /lib64/libz.so.1.2.11 %{buildroot}/lib64/libz.so.1
-cp /lib64/libz.so.1.2.11 %{buildroot}/lib64/
-cp /lib64/pkgconfig/zlib.pc %{buildroot}/lib64/pkgconfig/zlib.pc
-cp /usr/include/zconf.h %{buildroot}/usr/include/
-cp /usr/include/zlib.h %{buildroot}/usr/include/
-cp /share/man/man3/zlib.3 %{buildroot}/share/man/man3/
+mkdir -p %{buildroot}/opt/awn/lib64/pkgconfig
+mkdir -p %{buildroot}/opt/awn/usr/include
+mkdir -p %{buildroot}/opt/awn/share/man/man3/
+mkdir -p %{buildroot}/etc/ld.so.conf.d
+cp /lib64/libz.a %{buildroot}/opt/awn/lib64/
+ln -s /lib64/libz.so.1.2.11 %{buildroot}/opt/awn/lib64/libz.so
+ln -s /lib64/libz.so.1.2.11 %{buildroot}/opt/awn/lib64/libz.so.1
```

Comment:
Will these still link correctly? Correct me if I'm wrong, but wouldn't this cause `/opt/awn/lib64/libz.so` to link to a file that doesn't necessarily exist at `/lib64/libz.so.1.2.11` on install?